modules/solid_mechanics/test/tests/power_law_creep/tests

```diff
@@ -127,4 +127,11 @@
       detail = 'calculate a perfect Jacobian with the small strain formulation'
     []
   []
+  [composite_power_law_creep]
+    type = 'Exodiff'
+    input = 'composite_power_law_creep.i'
+    exodiff = 'composite_power_law_creep_out.e'
+    detail = 'power law creep that works with multiple phases.'
+    issues = '#28368'
+  []
```

Comment:
I'd like to see more regression tests here. In addition to the two suggested above, include:
- a test to make sure restart works
- a test to make sure this class works with plasticity
- a test to make sure this class works with small total strain
- tests for all of the new error messages added to the class